seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kafka/KafkaFormatIT.java

```diff
@@ -0,0 +1,731 @@
+        Startables.deepStart(Stream.of(MYSQL_CONTAINER)).join();
+        LOG.info("Mysql Containers are started");
+
+        LOG.info("The third stage: Starting Debezium Connector containers...");
+        createDebeziumContainer();
+        Startables.deepStart(Stream.of(DEBEZIUM_CONTAINER)).join();
+        LOG.info("Debezium Containers are started");
+
+        LOG.info("The third stage: Starting Canal containers...");
+        createCanalContainer();
+        Startables.deepStart(Stream.of(CANAL_CONTAINER)).join();
+        LOG.info("Canal Containers are started");
+
+        LOG.info("The fourth stage: Starting PostgreSQL container...");
+        createPostgreSQLContainer();
+        Startables.deepStart(Stream.of(POSTGRESQL_CONTAINER)).join();
+        Class.forName(POSTGRESQL_CONTAINER.getDriverClassName());
+        LOG.info("postgresql Containers are started");
+
+        given().ignoreExceptions()
+                .await()
+                .atLeast(100, TimeUnit.MILLISECONDS)
+                .pollInterval(500, TimeUnit.MILLISECONDS)
+                .atMost(2, TimeUnit.MINUTES)
+                .untilAsserted(this::initializeJdbcTable);
+
+        given().ignoreExceptions()
+                .atLeast(100, TimeUnit.MILLISECONDS)
+                .pollInterval(500, TimeUnit.MILLISECONDS)
+                .atMost(180, TimeUnit.SECONDS)
+                .untilAsserted(this::initKafkaConsumer);
+
+        LOG.info("start init Mysql DDl...");
+        inventoryDatabase.createAndInitialize();
+        LOG.info("end init Mysql DDl...");
+
+        // local file ogg data send kafka
+        given().ignoreExceptions()
+                .atLeast(100, TimeUnit.MILLISECONDS)
+                .pollInterval(500, TimeUnit.MILLISECONDS)
+                .atMost(3, TimeUnit.MINUTES)
+                .untilAsserted(this::initOggDataToKafka);
+
+        // debezium configuration information
+        Container.ExecResult extraCommand =
+                DEBEZIUM_CONTAINER.execInContainer(
+                        "bash",
+                        "-c",
+                        "cd /tmp/seatunnel/plugins/Jdbc && curl -i -X POST -H \"Accept:application/json\" -H  \"Content-Type:application/json\" http://"
+                                + getLinuxLocalIp()
+                                + ":8083/connectors/ -d @register-mysql.json");
+        Assertions.assertEquals(0, extraCommand.getExitCode());
+        // ensure debezium has handled the data
+        Thread.sleep(40 * 1000);
+        Awaitility.given()
+                .ignoreExceptions()
+                .atLeast(100, TimeUnit.MILLISECONDS)
+                .pollInterval(500, TimeUnit.MILLISECONDS)
+                .atMost(3, TimeUnit.MINUTES)
+                .untilAsserted(this::updateDebeziumSourceTableData);
+    }
+
+    @TestTemplate
+    public void testFormatCheck(TestContainer container) throws IOException, InterruptedException {
```

Comment:
I tested it and felt that I did not find a timeout, the same type of tasks are running on a container, I think to avoid the startup destruction process between different containers to save some time, if later found some timeouts I think we can increase the timeout setting,kafka e2e time can at least save more than 70% of the time

Reply:
Subsequent new Feature formats will increase the e2e time more. If each format starts an e2e class test, conflicts are more likely to occur